.circleci/config.yml

```diff
@@ -27,11 +27,11 @@ jobs:
     steps:
       - add_ssh_keys:
           fingerprints:
-            ## hashicorp-ci github "machine" user, which allows go to fetch
+            ## hc-github-team-waypoint github "machine" user, which allows go to fetch
             ## source for private modules.
             ## hostname must be set to "github.com":
             ## https://circleci.com/docs/2.0/gh-bb-integration/#creating-a-github-user-key
-            - "c6:96:98:82:dc:04:6c:39:dd:ac:83:05:e3:15:1c:98"
+            - "76:94:15:c8:a0:41:f0:8b:9b:f1:f7:36:4c:d5:7f:57
```

Comment:
```suggestion
            - "76:94:15:c8:a0:41:f0:8b:9b:f1:f7:36:4c:d5:7f:57"
```

Reply:
Ah you beat me to it :) committed

Reply:
I had just sat down with my tea so I AM READY 😆